### PR TITLE
#163083376 Changed status code on test

### PR DESCRIPTION
**Pull request by tolumide-ng**
#### What does this PR do?
Changes the status code on questions test.

#### Description of Task to be completed?
lauch a post request to create questions.

#### How should this be manually tested?
clone the repo, cd into it and **RUN** npm run test

#### Any background context you want to provide?
None

#### What are the relevant pivotal tracker stories?
[163083376](https://www.github.com/tolumide-ng/AndelaBootcamp/tree/bg-solved-test-issue-163083376) 

#### Screenshots (if appropriate)
image

### CODE REVIEW

File: server/controllers/questionsControllers.js [hunk 1]
```diff
@@ -7,25 +7,24 @@ const Question = {
     const data = req.body;
```
Comment (ludralph) [inline server/controllers/questionsControllers.js]: consider renaming the variable data. Variable names should be descriptive and meaningful. It should also be indicative of what is being stored

File: server/controllers/questionsControllers.js [hunk 2]
```diff
@@ -7,25 +7,24 @@ const Question = {
...
     const theUser = usersModels.signUsers.find(user => user.userId === data.user);
```
Comment (ludralph) [inline server/controllers/questionsControllers.js]: The variable name theUser doesn't tell me much about what is being stored. You can read this blog post for additional insight. https://medium.com/coding-skills/clean-code-101-meaningful-names-and-functions-bf450456d90c

**Review by ludralph — CHANGES_REQUESTED** [review]

File: server/controllers/questionsControllers.js (hunk 2 above)
Comment (tolumide-ng) [reply server/controllers/questionsControllers.js]: Thank you for the blog post. Would implement the lessons learned now.

**Review by ludralph — APPROVED** [review]
LGTM

**Pull request merged**